ScriptServiceUtil not initialized yet on startup

```
2017-05-31 20:09:57.118 [ERROR] [xbase.resource.BatchLinkableResource] - resolution of uriFragment '|::0.2.1.2.0.1.2.2.2.0.6.6.6::0::/1' failed.
java.lang.IllegalStateException: ScriptServiceUtil not initialized yet!
        at org.eclipse.smarthome.model.script.ScriptServiceUtil.getInstance(ScriptServiceUtil.java:88)
        at org.eclipse.smarthome.model.script.ScriptServiceUtil.getActionServices(ScriptServiceUtil.java:114)
        at org.eclipse.smarthome.model.script.internal.engine.ServiceTrackerActionServiceProvider.get(ServiceTrackerActionServiceProvider.java:21)
        at org.eclipse.smarthome.model.script.internal.engine.ServiceTrackerActionServiceProvider.get(ServiceTrackerActionServiceProvider.java:1)
        at org.eclipse.smarthome.model.script.scoping.ScriptImplicitlyImportedTypes.getActionClasses(ScriptImplicitlyImportedTypes.java:100)
        at org.eclipse.smarthome.model.script.scoping.ScriptImplicitlyImportedTypes.getStaticImportClasses(ScriptImplicitlyImportedTypes.java:94)
        at org.eclipse.xtext.xbase.scoping.batch.ImplicitlyImportedFeatures.getStaticImportClasses(ImplicitlyImportedFeatures.java:66)
        at org.eclipse.xtext.xbase.scoping.batch.XbaseBatchScopeProvider.newSession(XbaseBatchScopeProvider.java:112)
        at org.eclipse.xtext.xbase.typesystem.internal.DefaultReentrantTypeResolver.resolve(DefaultReentrantTypeResolver.java:162)
        at org.eclipse.xtext.xbase.typesystem.internal.DefaultReentrantTypeResolver.reentrantResolve(DefaultReentrantTypeResolver.java:139)
        at org.eclipse.xtext.xbase.typesystem.internal.CachingBatchTypeResolver$LazyResolvedTypes.resolveTypes(CachingBatchTypeResolver.java:80)
        at org.eclipse.xtext.xbase.typesystem.internal.CachingBatchTypeResolver$2.process(CachingBatchTypeResolver.java:57)
        at org.eclipse.xtext.xbase.typesystem.internal.CachingBatchTypeResolver$2.process(CachingBatchTypeResolver.java:1)
        at org.eclipse.xtext.util.concurrent.IUnitOfWork$Void.exec(IUnitOfWork.java:37)
        at org.eclipse.xtext.util.OnChangeEvictingCache.execWithoutCacheClear(OnChangeEvictingCache.java:129)
        at org.eclipse.xtext.xbase.typesystem.internal.CachingBatchTypeResolver.doResolveTypes(CachingBatchTypeResolver.java:53)
        at org.eclipse.xtext.xbase.typesystem.internal.AbstractBatchTypeResolver.resolveTypes(AbstractBatchTypeResolver.java:69)
        at org.eclipse.xtext.xbase.resource.BatchLinkingService.resolveBatched(BatchLinkingService.java:60)
        at org.eclipse.xtext.xbase.resource.BatchLinkingService.resolveBatched(BatchLinkingService.java:41)
        at org.eclipse.xtext.xbase.resource.BatchLinkableResource.getEObject(BatchLinkableResource.java:117)
        at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.getEObject(ResourceSetImpl.java:223)
        at org.eclipse.emf.ecore.util.EcoreUtil.resolve(EcoreUtil.java:203)
        at org.eclipse.emf.ecore.util.EcoreUtil.resolve(EcoreUtil.java:259)
        at org.eclipse.emf.ecore.impl.BasicEObjectImpl.eResolveProxy(BasicEObjectImpl.java:1477)
        at org.eclipse.xtext.xbase.impl.XAbstractFeatureCallImplCustom.getFeature(XAbstractFeatureCallImplCustom.java:48)
        at org.eclipse.xtext.xbase.validation.XbaseValidator.checkDelegateConstructorIsFirst(XbaseValidator.java:927)
        at sun.reflect.GeneratedMethodAccessor13.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[:1.8.0_131]
        at java.lang.reflect.Method.invoke(Method.java:498)[:1.8.0_131]
        at org.eclipse.xtext.validation.AbstractDeclarativeValidator$MethodWrapper.invoke(AbstractDeclarativeValidator.java:118)[142:org.eclipse.xtext:2.9.2]
        at org.eclipse.xtext.validation.AbstractDeclarativeValidator.internalValidate(AbstractDeclarativeValidator.java:312)[142:org.eclipse.xtext:2.9.2]
        at org.eclipse.xtext.validation.AbstractInjectableValidator.validate(AbstractInjectableValidator.java:71)[142:org.eclipse.xtext:2.9.2]
        at org.eclipse.xtext.validation.CompositeEValidator.validate(CompositeEValidator.java:151)[142:org.eclipse.xtext:2.9.2]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidate(Diagnostician.java:171)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:158)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:181)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:181)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:181)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:181)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:185)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:185)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:185)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:185)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:185)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.doValidateContents(Diagnostician.java:185)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:161)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:137)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.emf.ecore.util.Diagnostician.validate(Diagnostician.java:108)[65:org.eclipse.emf.ecore:2.11.1.v20150805-0538]
        at org.eclipse.smarthome.model.core.internal.ModelRepositoryImpl.validateModel(ModelRepositoryImpl.java:265)[123:org.eclipse.smarthome.model.core:0.9.0.201705301617]
        at org.eclipse.smarthome.model.core.internal.ModelRepositoryImpl.addOrRefreshModel(ModelRepositoryImpl.java:90)[123:org.eclipse.smarthome.model.core:0.9.0.201705301617]
        at org.eclipse.smarthome.model.core.internal.folder.FolderObserver.checkFile(FolderObserver.java:244)[123:org.eclipse.smarthome.model.core:0.9.0.201705301617]
        at org.eclipse.smarthome.model.core.internal.folder.FolderObserver.notifyUpdateToModelRepo(FolderObserver.java:170)[123:org.eclipse.smarthome.model.core:0.9.0.201705301617]
        at org.eclipse.smarthome.model.core.internal.folder.FolderObserver.updated(FolderObserver.java:148)[123:org.eclipse.smarthome.model.core:0.9.0.201705301617]
        at org.apache.felix.cm.impl.helper.ManagedServiceTracker.updated(ManagedServiceTracker.java:189)[3:org.apache.felix.configadmin:1.8.12]
        at org.apache.felix.cm.impl.helper.ManagedServiceTracker.updateService(ManagedServiceTracker.java:152)[3:org.apache.felix.configadmin:1.8.12]
        at org.apache.felix.cm.impl.helper.ManagedServiceTracker.provideConfiguration(ManagedServiceTracker.java:85)[3:org.apache.felix.configadmin:1.8.12]
        at org.apache.felix.cm.impl.ConfigurationManager$ManagedServiceUpdate.provide(ConfigurationManager.java:1461)[3:org.apache.felix.configadmin:1.8.12]
        at org.apache.felix.cm.impl.ConfigurationManager$ManagedServiceUpdate.run(ConfigurationManager.java:1417)[3:org.apache.felix.configadmin:1.8.12]
        at org.apache.felix.cm.impl.UpdateThread.run0(UpdateThread.java:141)[3:org.apache.felix.configadmin:1.8.12]
        at org.apache.felix.cm.impl.UpdateThread.run(UpdateThread.java:109)[3:org.apache.felix.configadmin:1.8.12]
        at java.lang.Thread.run(Thread.java:748)[:1.8.0_131]

```

See https://community.openhab.org/t/snapshot-930-not-starting-properly/29252

--- a/bundles/model/org.eclipse.smarthome.model.rule.runtime/OSGI-INF/runtime.xml
+++ b/bundles/model/org.eclipse.smarthome.model.rule.runtime/OSGI-INF/runtime.xml
@@ -13,4 +13,5 @@
    <service>
       <provide interface="org.eclipse.smarthome.model.core.ModelParser"/>
    </service>
+   <reference cardinality="1..1" interface="org.eclipse.smarthome.model.script.ScriptServiceUtil" name="ScriptServiceUtil" policy="static"/>
 </scr:component>

